### ISSUE TITLE
Plain mode

### ISSUE THREAD

**Issue by marco-trovato** [giuspen/cherrytree#2123]
Please add a mode to simply use folders/directories for hierarchy, and files for text content.
(Instead of saving everything to a single file)

**Benefits:**

This simple move could make CherryTree the best outliner, because right now every Outliner software is using its own format, but saving like this would make the format future-proof.

**"The EXPORT function exists already"**

I'm not talking about EXPORTING, but working directly with files and subdirectories (i.e.: search by grep and find)

**Why?**

Right now there is no way to open any outliner file on Android. That means sending yourself a memo every time to add stuff when you will be at the computer on the Outliner. But files would be immediately available everywhere, and also on future platforms.

**Comment by giuspen** [comment]
I have in mind to support such non single file approach in future

**Comment by mahdi1234** [comment]
Sorry for ping @giuspen but I'd like to ask whether this is somewhere on the roadmap.

The things is while I really love cherrytree and been using it for like a decade it's becoming real pain to sync it between multiple devices (I have 4 computers home/work) and I have to remember always close the app to not have conflicts. Being able to convert existing single file into dir structure would solve this problem. If you don't plan it at all that's fine, but I'd like to know whether I should start thinking about looking elsewhere or there is some hope :) Thanks.

**Comment by marco-trovato** [comment]
@mahdi1234 my suggestion is to use [Obsidian](https://obsidian.md) in the meanwhile.
I gave up and unfortunately this is what I'm doing right now.

**Comment by mahdi1234** [comment]
@marco-trovato Did you try/succeeded migrate tags? I have 1.5K notes (on my bigger file; and 500 on work profile) and it seems Obsidian is okish, but I'd like to stay with cherrytree if there's some chance of getting this done as otherwise it's fantastic app. So thus asking whether it's on roadmap.

**Comment by marco-trovato** [comment]
Trust me. I have used TreePad and every single alternative ([KeyNote NF](
https://github.com/dpradov/keynote-nf) is the most feature complete), for
several decades. Emacs orgmode, Jreepad, etc.

Trust me. Use plain text files. Transcend.

You can preview them with Midnight Commander on the other pane, and grep
them recursively from the menu even from SSH from your mobile phone on the
go, instead of having to get to your computer with your specific Outliner
software that only works on that OS and not on your phone, procrastinating
the transcription from other temporary places for "later".

Nothing is better than textfiles because you can open them everywhere for
the years to come. Just use directories as hierarchy. Any OS will copy/zip
1.5k files in just few seconds.

Transcend my friend.

```
while [ 1 ]
do

 SELECTED=$(ls -a -I . | fzf --layout reverse --preview="if [ -d {} ] ;
then tree {} ; else cat {} ; fi" --preview-window=bottom:50%:wrap)

 if [ ! -e "$SELECTED" ]; then
  (return 0 2>/dev/null) && return || exit
 fi

 if [ -d "$SELECTED" ]; then
  cd "$SELECTED"
 else
  $EDITOR "$SELECTED"
 fi

done
```

**Comment by marco-trovato** [comment]
Trust me. I have used TreePad and every single alternative ([KeyNote NF](
https://github.com/dpradov/keynote-nf) is the most feature complete), for several decades. Emacs orgmode, Jreepad, etc.

Trust me. Use plain text files. Transcend.

You can preview them with Midnight Commander on the other pane, and grep them recursively from the menu even from SSH from your mobile phone on the
go, instead of having to get to your computer with your specific Outliner software that only works on that OS and not on your phone, procrastinating the transcription from other temporary places for "later".

Nothing is better than textfiles because you can open them from everywhere for the years to come. Just use directories as hierarchy. Any OS will copy/zip
1.5k files in just few seconds.

Transcend my friend.


```
while [ 1 ]
do

	SELECTED=$(ls -a -I . | fzf --layout reverse --preview="if [ -d {} ] ; then tree {} ; else cat {} ; fi" --preview-window=bottom:50%:wrap)

	if [ ! -e "$SELECTED" ]; then
		(return 0 2>/dev/null) && return || exit
	fi

	if [ -d "$SELECTED" ]; then
		cd "$SELECTED"
	else
		$EDITOR "$SELECTED"
	fi

done

```

**Comment by giuspen** [comment]
@mahdi1234 this is actually high priority, while it will probaly not be completed in 0.99.55, it will almost certainly be released in 0.99.56

**Comment by giuspen** [comment]
This has been implemented and while still under testing will be in 0.99.56
